src/Services/FHIR/FhirEncounterService.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace OpenEMR\Services;
```

Comment:
namespace for your new service location
namespace OpenEMR\Services\FHIR

do same for all fhir services moved...

Reply:
Done! Thanks For pointing it :)